### ISSUE TITLE
End mission when revive and respawn are no longer possible for any players

### ISSUE THREAD

**Issue by kcgrimes** [kcgrimes/grimes-simple-revive#49]
Per runforrest, ref https://forums.bohemia.net/forums/topic/167673-grimes-simple-revive-script/?page=7&tab=comments#comment-2886312

end mission when all players are unconscious

**Comment by kcgrimes** [comment]
Raised this one keeping an AI-integrated future in mind. End mission when revive and respawn are no longer possible, meaning there is no way for any player to walk again, and therefore all players are doing nothing but running down the clock.

**Comment by kcgrimes** [comment]
Run a loop that checks every 5-10 seconds or so on the server only. If certain conditions are achieved that indicate a player will never walk again, then execute BIS_fnc_endMissionServer which will correctly end the mission for everyone.

**Comment by kcgrimes** [comment]
Done, with an option to enable/disable the feature in the settings. Only "catch" is that it is based on players either fighting each other or fighting AI, but not both.

Ex 1
WEST (players), EAST (players), INDE (AI) are all in a 3-way free-for-all. 
WEST loses. Because WEST lost, EAST wins. INDE, which is AI only, is not considered.

Ex 2
WEST (players), EAST (players), INDE (players), CIV (AI) are in a TDM where EAST and INDE are allied in a 3-way against WEST and CIV. 
WEST loses. Because WEST lost, EAST and INDE wins, and CIV, which is AI only, is not considered. 
In this example, if EAST and INDE were not allies, the fight would continue.

Basically, after one side loses, if a remaining side of players is not enemies with another side of players, the game will also end with the remaining side of players. Suggested workaround: Disable this feature.
